lib/round.rb

```diff
@@ -0,0 +1,51 @@
+class Round
+    attr_reader :deck, :turns, :current_card, :card_index
```

Comment:
We don't need a `current_card` `attr_reader` because you have defined that method below.